keanu-project/src/main/java/io/improbable/keanu/util/dot/VertexDotLabel.java

```diff
@@ -0,0 +1,66 @@
+package io.improbable.keanu.util.dot;
+
+import io.improbable.keanu.vertices.ConstantVertex;
+import io.improbable.keanu.vertices.Vertex;
+
+import java.util.Objects;
+
+public class VertexDotLabel {
+    private Vertex vertex;
+    private String value = "";
+    private String annotation = "";
+    private String vertexLabel = "";
+    private static final String DOT_LABEL_OPENING = "[label=\"";
+    private static final String DOT_LABEL_CLOSING = "\"]";
+
+    public VertexDotLabel(Vertex vertex) {
+        this.vertex = vertex;
+    }
+
+    public enum VertexDotLabelType{
+        VALUE, CLASS_NAME, ANNOTATION_LABEL, VERTEX_LABEL
+    }
+
+    public void setDotLabel(VertexDotLabelType labelType, String label) {
```

Comment:
I think I had intended to use it more, but ended up not doing it. I agree that it would be more readable to set value etc explicitly, but why is it simpler and safer?